salt/modules/shadow.py

```diff
@@ -379,21 +386,31 @@ def set_password(name, password, use_usermod=False, root=None):
         if not os.path.isfile(s_file):
             return ret
         lines = []
+        user_found = False
+        lstchg = six.text_type(
```

Comment:
I don't think so. If you're developing for a *NIX platform and do not understand what the UNIX epoch is...

Reply:
🤷‍♂️ while I agree with that sentiment, I also agree that explicit is better than implicit. I always encourage the latter. Especially since future me is usual the one maintaining the software I wrote and he always appreciates it 🤣